### PR TITLE
 Team subscribe badging visual changes suggested by Cecile

### PR DESCRIPTION
**Pull request by cjb**
@keybase/react-hackers CC @cecileboucheron 

* orange -> blue

* have the badge positioned on top of the gear icon, top right edge

* move badge on popup menu from left side to right side

### CODE REVIEW

File: shared/chat/inbox/row/big-team-header/index.js [hunk 1]
```diff
@@ -27,19 +27,15 @@ class BigTeamHeader extends React.PureComponent<Props> {
         <Text type="BodySmallSemibold" style={teamStyle}>
           {props.teamname}
         </Text>
-        <Icon
-          className="icon"
-          type="iconfont-gear"
+        <ClickableBox
           onClick={isMobile ? () => props.onClickGear() : props.onClickGear}
```
Comment (chrisnojima) [inline shared/chat/inbox/row/big-team-header/index.js]: Why is this necessary?

Comment (cjb) [reply shared/chat/inbox/row/big-team-header/index.js]: Just so that taps that hit the blue badge box rather than the gear icon also fire the onclick.  Make sense?

Comment (chrisnojima) [reply shared/chat/inbox/row/big-team-header/index.js]: But both call the same props and it's an arrow closure so the second case just works right

Comment (cjb) [reply shared/chat/inbox/row/big-team-header/index.js]: Ah, thought you were asking about the ClickableBox.  Surprisingly, it's necessary:
```js
const mapDispatchToProps = (dispatch, {teamname}) => ({
  onClickGear: (evt?: SyntheticEvent<Element>) =>
    dispatch(
      navigateAppend([
        {
          props: {
            teamname,
            isSmallTeam: false,
            position: 'bottom right',
            targetRect: evt && evt.currentTarget.getBoundingClientRect(),
          },
          selected: 'infoPanelMenu',
        },
      ])
    ),
})
```
So, we'll have an `evt` passed through by default, and on mobile we want to suppress that so that we don't try to call `getBoundingClientRect()` on it, which doesn't exist on mobile.

Seems like it'd be preferable to instead just test for `getBoundingClientRect`'s existence inside the dispatch func and call it if it's there?  CC @buoyad, this is relative popup stuff.

**Comment by chrisnojima** [comment]
Is advocate for the relative hoc to do the right thing and just ignore it
in mobile. Either use isMobile itself or check for bounding rect. I want to
reduce how much boilerplate stuff like this callers need to do

**Comment by buoyad** [comment]
We do this in different ways all over now, in some places we do what @cjb has here. In others we pass down the event regardless and use `isMobile` in dispatch to avoid calling the rect function. As it’s implemented the boilerplate is required in one place or another. If we transition this to the new one it’ll be more like you describe @chrisnojima.

**Comment by cjb** [comment]
Thanks!  Just pushed a commit with a mild improvement, will leave a larger cleanup to @buoyad's transition.
